src/adapters/notificationAdapter.ts

```
@@ -0,0 +1,14 @@
+export default function notificationAdapter(notification: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)